t/lib/Bakesale/Context.pm

```diff
@@ -51,4 +51,25 @@ package Bakesale::Context::System {
   with 'Ix::Context';
 }
 
+package Bakesale::Context::NoAuth {
+  use Moose;
+
+  sub code { 401 }
+
+  with 'Ix::Context::Error';
+}
+
```

Comment:
I figure most cases will look like the above - we want to return a specific http response code and nothing more